CMake/External_ZeroMQ.cmake

```diff
@@ -0,0 +1,21 @@
+fletch_Require_Make()
```

Comment:
Again, zeroMQ does have a CMakeLists.txt file. We should use that rather than configure/make.